src/rust/ide/src/controller/graph/executed.rs

```diff
@@ -45,7 +45,7 @@ pub enum Notification {
     /// being updated.
     ComputedValueInfo(model::execution_context::ComputedValueExpressions),
     /// Notification emitted when the node has been entered.
-    EnteredNode(double_representation::node::Id),
+    EnteredNode(double_representation::node::Id,MethodPointer),
```

Comment:
So we have two structures named `LocalCall` that wrap these two values with proper semantics.
Please, please tell me that we can use one of them here instead of raw pair.